script/Arcueid/Day 12/12_00_ARC12_2.txt

```diff
@@ -351,42 +357,43 @@ In a moment, I'll be able to see his face clearly.
 [sha:e4fe9c893830d1e4f04dbc7a14fd4d46399696bf]{
 -- Page 55, Offset 30665.
 -- 誰に似ているのか。
-Do I know him?
+Who does he remind me of?
 }
 [sha:e4fe9c893830d1e4f04dbc7a14fd4d46399696bf]{
 -- Page 55, Offset 30666.
 -- 誰に似ているのか。
-Do I know him?
+Who does he remind me of?
 }
 [sha:e4fe9c893830d1e4f04dbc7a14fd4d46399696bf]{
 -- Page 55, Offset 30667.
 -- 誰に似ているのか。
-Do I know him?
+Who does he remind me of?
 }
 [sha:547e8af76558eaee0267b9cbe818762115ce2c7d]{
 -- Page 55, Offset 30668.
 -- 自分は、誰を忘れているのか――――――
-Is he someone that I've forgotten――――――
+Who is it that I've forgotten――――――
 }
 [sha:9429372c88b7b16f2e9f98c6919a76c0035ab879]{
 -- Page 56, Offset 30669.
 -- 俺に合わせるように、男の足が止まった。
-The man stops to face me.
+The man stops, as if to greet me.
 }
 [sha:7e777afd2448eb07a2fa781e14c5e95100fa5210]{
 -- Page 56, Offset 30670.
 -- まだ遠い。赤い斜陽で貌も見えない。
-He's still far. I can't see his face from the red sunset.
+He's still a ways away.
+ I can't make out his face, backlit as it is by the red light of the setting sun.
```

Comment:
roa is col. kilgore?

Reply:
I love the smell of black keys in the morning